<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 823, in lint_file
    recipe = _recipe.Recipe.from_file(
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/render/recipe.py", line 245, in from_file
    raise exc from exc
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/render/recipe.py", line 236, in from_file
    recipe._load_from_string(text.read())
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/render/recipe.py", line 172, in _load_from_string
    self.render()
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/render/recipe.py", line 301, in render
    self.meta = renderer_utils.render(self.recipe_file, self.dump(), self.selector_dict, self.renderer)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/render/_renderer.py", line 256, in render
    raise JinjaRenderFailure(recipe_file, message=str(exc)) from exc
percy.render.exceptions.JinjaRenderFailure: (PosixPath('/tmp/abs_a70suqvwdl/clone/recipe/meta.yaml'), "render.<locals>.<lambda>() got an unexpected keyword argument 'exact'")

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/run.py", line 208, in main
    return_code, report = execute_linter(
                          ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/run.py", line 180, in execute_linter
    result = linter.lint(recipes, subdir, variant_config_files, exclusive_config_files, fix_flag)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 725, in lint
    msgs = self.lint_file(
           ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 840, in lint_file
    recipe = _recipe.Recipe(recipe_name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/render/recipe.py", line 109, in __init__
    self.recipe_dir = recipe_file.parent
                      ^^^^^^^^^^^^^^^^^^
AttributeError: 'str' object has no attribute 'parent'

ERROR conda.cli.main_run:execute(125): `conda run conda-lint /tmp/abs_a70suqvwdl/clone` failed. (See above for error)
</details>

--- a/abs.yaml
+++ b/abs.yaml
@@ -1,1 +1,4 @@
 aggregate_check: false
+
+channels:
+  - https://staging.continuum.io/prefect/fs/libtiff-feedstock/pr21/e3ccbbe

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -27,7 +27,7 @@ source:
     - patches/0018-Use-sed-from-system.patch
 
 build:
-  skip: true  # [win]
+  skip: true  # [win or not (linux and x86_64)]
   merge_build_host: true  # [win]
   number: {{ build_number }}
   rpaths:
@@ -200,6 +200,28 @@ test:
     - if not exist %PREFIX%\\Lib\\R\\bin\\x64\\R.lib exit 1  # [win]
   files:
     - test-svg.R
+  downstreams:
+    r-matrix  # 0 dependencies, compiled
+    r-rcpp  # 0 dependencies, compiled
+    r-data.table  # 0 dependencies, compiled
+    r-stringi  # 0 dependencies, compiled
+    r-ijtiff  # 1 dependency, compiled (uses libtiff)
+    r-ragg  # 1 dependency, compiled (uses libtiff)
+    r-gert  # 1 dependency, compiled
+    r-dplyr  # 2 dependencies, compiled
+    r-igraph  # 2 dependencies, compiled
+    r-rgl  # 2 dependencies, compiled
+    r-svglite  # 3 dependencies, compiled
+    r-patchwork  # 3 dependencies, noarch
+    r-ggplotify  # 4 dependencies, noarch
+    r-tidytree  # 5 dependencies, noarch
+    r-nlme  # 2 dependencies, compiled
+    r-caret  # 3 dependencies, compiled
+    r-purrr  # 3 dependencies, compiled
+    r-stringi  # 0 dependencies, compiled
+    r-readr  # 4 dependencies, compiled
+    r-mgcv  # 4 dependencies, compiled
+    r-survival  # 3 dependencies, compiled
 
 about:
   home: https://www.r-project.org/

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -154,7 +154,8 @@ requirements:
     - curl                               # [not win]
     - {{native}}bzip2                    # [win]
     - {{native}}libjpeg-turbo            # [win]
-    - {{native}}libiconv                 # [win]
+    # lib/R/lib/libR.so requires libiconv at runtime
+    - {{native}}libiconv
     - {{native}}gmp                      # [win]
     - {{native}}fftw                     # [win]
     - {{native}}xz                       # [win]

